### PR TITLE
add local running

### PR DESCRIPTION
**Pull request by SajjadSabzkar**
Fix issue #3

### CODE REVIEW

**Comment by amiremohamadi** [comment]
lgtm. just a few comments.

File: Makefile [hunk 1]
```diff
@@ -18,3 +18,11 @@ config-upload: ## upload config to kv storage. "name" arg should be provided
 .PHONY: prepare
 prepare: ## prepare developing environment
 	@ sed -i -e "s/KV_BINDING/$(KV_BINDING)/" -e "s/KV_ID/$(KV_ID)/" wrangler.toml
+
+.PHONY: dev-config
+dev-config: ## write config locally
+	@ npx wrangler kv:key put CONFIG --path=$(name) --binding $(KV_BINDING) --local
```
Comment (amiremohamadi) [inline Makefile:24]: nice approach! thanks! :beer:

Comment (SajjadSabzkar) [reply Makefile:24]: That's one of my challenge.

File: README.md [hunk 2]
```diff
@@ -32,3 +32,25 @@ $ make config-upload
...
+3.  to run wiseass locally:
+```sh
+$ make dev
+```
+4. install [cloudflared](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/get-started/create-local-tunnel/) and run the following command to expose your local wiseass instance to the internet.
+
+ 5. set a telegram webhook:
+ https://api.telegram.org/bot{YOUR_BOT_TOKEN}/setWebhook?url=https://{CLOUDFLARED_EXPOSED_BASE_URL}/updates 
+```sh
+$ cloudflared tunnel --url http://localhost:8780
```
Comment (amiremohamadi) [inline README.md:55]: i think this should be moved to L51.

**Comment by amiremohamadi** [comment]
would u please squash the commits into one with a meaningful message?

**Comment by SajjadSabzkar** [comment]
> would u please squash the commits into one with a meaningful message?

Sure!

**Pull request merged**